src/profile/profile.controller.ts

```diff
@@ -0,0 +1,20 @@
+import { Controller, Post, Body, Get, ParseUUIDPipe, Param} from '@nestjs/common';
+import { CreateProfileDto } from './dto/create-profile.dto';
+import { ProfileService } from './profile.service';
+
+@Controller('profile')
+export class ProfileController {
+    constructor(private profileService: ProfileService) {}
+
+    @Post('/create')
+    async createProfile(@Body() body: CreateProfileDto) {
+        const userProfile = await this.profileService.createprofile(body)
+        return userProfile;
+    }
+
+    @Get('/:id')
```

Comment:
![image](https://user-images.githubusercontent.com/84224449/186766427-7aa37b23-5722-49e8-97e5-d98eaf52a409.png)

This is how your controllers should look like, Compare the differences .

Reply:
You have to take my word that is working😅😅
 ... currently having a hard time migrating, the migration just keeps pushing a migration table to the DB and not the entities table.

And for the second one... You mentioned in one of my last pushes that you don't use **ZaLaResponse** in the controller hence why I didn't add that

Reply:
you are misunderstanding him.. don't use ZaLaResponse in services but controllers.. you only use ZaLaResponse in services when you are trying to catch errors

Reply:
Ohhh I think I get what you guys are trying to say... I will fix that now 

Thank you

Reply:
What I get any time I run a migration

![image](https://user-images.githubusercontent.com/36968337/186836724-30b03554-4940-4275-99e6-de25eda76952.png)

And this is what appears in my DB

![image](https://user-images.githubusercontent.com/36968337/186836890-42ae178f-f00f-4d00-a0c2-454b045dd6ca.png)

Reply:
If you have an issue, you ask for help...Don't go ahead to push code that  is potentially buggy.

Reply:
Help ... 
🦸🏾‍♂️

I can't seem to run migration

Reply:
Add migration file with: npm run typeorm migration:generate ./src/database/migration/${file name}

Then: npm run migration

But before then ensure your env and db is properly setup

Reply:
exactly... change the ${file name} to the name of the migration file you want to generate
...

Reply:
Thank you, guys... y'all just saved me lots of headaches